src/xdocs/config_whitespace.xml

```diff
@@ -16,6 +16,266 @@
       </macro>
     </section>
 
+    <section name="ChainedMethodCallWrap">
+      <p>Since Checkstyle 10.2</p>
+      <subsection name="Description" id="ChainedMethodCallWrap_Description">
+        <p>
+          Checks whether method calls specified by identifier pattern are wrapped.
+          Line wrapping means to insert line breaks in code to make it more readable and
+          adhere to line length standards.
+        </p>
+        <p>
+          Rationale: Long chained method calls are difficult to read and comprehend and sometimes
```

Comment:
Please add a comma here: `Rationale: Long chained method calls are difficult to read and comprehend, and...`

Reply:
Done.

Reply:
Resolved.